cypress/integration/admin/account/account.spec.js

```diff
@@ -26,4 +26,28 @@ context("Account Module", () => {
             cy.should("exist");
         });
     });
+
+    it("should be able to change avatar", () => {
+        cy.visit("/account");
+        cy.get('div[data-role="select-image"]').click();
+
+        cy.findByTestId("fm-list-wrapper").dropFile("sample.jpeg", "image/jpeg");
+        cy.findByText("File upload complete.").should("exist");
+        cy.get('div[data-testid="fm-list-wrapper-file"]:first-child').click();
+
+        cy.get('div.mdc-layout-grid div > img').should('exist');
```

Comment:
Can we add a testid for this? I'm afraid that this could cause flakiness in the future.